packages/p2p/src/components/online-status/online-status-label.jsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { Text } from '@deriv/components';
+import { isMobile } from '@deriv/shared';
+import { localize } from 'Components/i18next';
+import { observer } from 'mobx-react-lite';
+import PropTypes from 'prop-types';
+import moment from 'moment';
+
+const OnlineStatusLabel = ({ is_online, last_online_time, size = isMobile() ? 'xxxs' : 'xs' }) => {
+    const last_online_label = () => {
+        if (!is_online) {
+            if (last_online_time) {
```

Comment:
Is it possible to combine these 2?

Reply:
We need 2 separate branching statements for this, since if `is_online` is truthy, we need to show 'Online', and if `last_online_time` is not nullish, we need to show 'Seen more than 6 months ago'